packages/server/customer-os-common-module/repository/postgres/postgres_tenant_webhook_repository.go

```diff
@@ -0,0 +1,64 @@
+package repository
+
+import (
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-common-module/repository/postgres/entity"
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-common-module/repository/postgres/helper"
+	"gorm.io/gorm"
+)
+
+type TenantWebhookRepo struct {
+	db *gorm.DB
+}
+
+type TenantWebhookRepository interface {
+	GetWebhook(tenant, event string) helper.QueryResult
+	GetWebhooks(tenant string) helper.QueryResult
+	SaveWebhook(integration entity.TenantWebhook) helper.QueryResult
+}
+
+func NewTenantWebhookRepo(db *gorm.DB) *TenantWebhookRepo {
+	return &TenantWebhookRepo{db: db}
+}
+
+func (r *TenantWebhookRepo) GetWebhook(tenant, event string) helper.QueryResult {
+	var webhookEntity entity.TenantWebhook
+	err := r.db.
+		Where("tenant_name = ?", tenant).
+		Where("event = ?", event).
+		First(&webhookEntity).Error
+
+	if err != nil {
+		return helper.QueryResult{Error: err}
+	}
+
+	return helper.QueryResult{Result: &webhookEntity}
+}
+
+func (r *TenantWebhookRepo) GetWebhooks(tenant string) helper.QueryResult {
+	var webhookEntity entity.TenantWebhook
+	err := r.db.
+		Where("tenant_name = ?", tenant).
+		Find(&webhookEntity).Error
+
+	if err != nil {
+		return helper.QueryResult{Error: err}
+	}
+
+	return helper.QueryResult{Result: &webhookEntity}
+}
+
+func (r *TenantWebhookRepo) SaveWebhook(webhook entity.TenantWebhook) helper.QueryResult {
+	webhookEntity := entity.TenantWebhook{
```

Comment:
this is missing the ID property to make the update

Reply:
it's save new, not update, should I add an update function too?